tools/build.sh

```diff
@@ -86,9 +86,25 @@ esac
 
 if [[ ${LANGUAGE} != en ]]; then
   # limit output to specified language
-  po4a --no-update po4a.cfg \
-    2> >(grep -v "unmatched end of environment .multicols\| (po4a::tex)$" >&2) \
-    | grep "/${LANGUAGE}/" || exit 1
+  if ! po4a --no-update po4a.cfg \
+       2> >(grep -v "unmatched end of environment .multicols\| (po4a::tex)$" >&2) \
+       | grep "/${LANGUAGE}/"; then
+
+    echo -e "---\npo4a failed for language ${LANGUAGE}, please fix the errors."
+
+    po_error_found=0
+    while IFS= read -r -d '' file; do
+      if ! msgfmt -c --check-format "$file" -o /dev/null 2>&1; then
```

Comment:
BTW, this should be the default behavior of po4a. It gives a meaningless error message, while it should give the output of this one.